### PR TITLE
Novo Technology Channel

### PR DESCRIPTION
**Pull request by johncordeiro**
This PR adds the [Novo Technology](http://www.novotechnologyinc.com/) SMS channel. NOVO Technology is an SMS Aggregator from the Republic of Trinidad and Tobago.

The channel was created following the documentation provided in direct conversations with Technical Team from the company.

### CODE REVIEW

**Comment by nicpottier** [comment]
Looks good to me, @norkans7 can you take a gander as well?

File: temba/channels/types/novo/tests.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
```
Comment (norkans7) [inline temba/channels/types/novo/tests.py]: We now support Python 3, we no longer need __future__ import and six library

File: temba/channels/types/novo/type.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    courier_url = r"^nv/(?P<uuid>[a-z0-9\-]+)/(?P<action>receive)$"
+
+    code = "NV"
+    category = ChannelType.Category.PHONE
+
+    name = "Novo"
+
+    claim_blurb = _(
+        """If you are based in Trinidad & Tobago, you can purchase a short code from <a href="http://www.novotechnologyinc.com/">Novo</a> and connect it in a few simple steps."""
+    )
```
Comment (nicpottier) [inline temba/channels/types/novo/type.py]: Pretty sure this replaces the claim template @norkans7

Comment (norkans7) [reply temba/channels/types/novo/type.py]: That is show on the section for the channel type when we have all types listed,

the claim template is for the form

Comment (norkans7) [reply temba/channels/types/novo/type.py]: Ohh right, the template is need for those that extends the AuthenticatedForm.

File: temba/channels/types/novo/type.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
+
+import six
```
Comment (norkans7) [inline temba/channels/types/novo/type.py]: I think six is not needed too

**Pull request merged**